cypress/e2e/class_management/createEditDeleteClass.feature

```diff
@@ -0,0 +1,61 @@
+@regression_test
+@stable_test
+@schedule_test
+Feature:  To create, edit and delete class on dBC or on BRB
+    # Note: On main-BRB student list is not loading on class management page, so test doesn't execute on brb.
```

Comment:
remove this line as it will be the issue why you are not seeing test in runner

Reply:
yes, this comment was not allowed below the feature line. Now I put it below the scenario as comment and works.